src/include/distributed/multi_copy.h

```diff
@@ -43,8 +47,54 @@ typedef struct NodeAddress
 	int32 nodePort;
 } NodeAddress;
 
+/* CopyDestReceiver can be used to stream results into a distributed table */
+typedef struct CitusCopyDestReceiver
+{
+	DestReceiver pub;
```

Comment:
This `pub` naming scheme seems taken from PostgreSQL's `COPY` code. How much else is modeled after that and how much is novel? Trying to figure out remaining places to apply most scrutiny.